__tests__/Unit/MainTest.php

```diff
@@ -0,0 +1,254 @@
+<?php
+//load SDK
+require_once(dirname(dirname(__DIR__)) . '/vendor/autoload.php');
+
+//test
+class MainTest extends \Codeception\Test\Unit
+{
+    protected $productsApiInstance;
+    protected $campaignsApiInstance;
+    protected $validationRulesApiInstance;
+    protected $customersApiInstance;
+    protected $qualificationsApiInstance;
+    protected $redemptionsApiInstance;
+    protected $stackedDiscountsApiInstance;
+
+    protected function _before()
```

Comment:
Daj tu może TODO, refactor. Czy coś, żeby było wiadomo, że to nie powinno tak być kiedyś.